public/login.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
I think it's conceptually easier to have a separate login file than make it an Angular 2 component. This way our Angular app can be focused on its domain (i.e. game instances, chat instance, and battles) and we don't have to worry about side effects of form posts.